test/gtest/ucp/test_ucp_tag_match.cc

```diff
@@ -490,6 +490,64 @@ class test_ucp_tag_match_rndv : public test_ucp_tag_match {
                                      tls, RNDV_SCHEME_GET_ZCOPY, result);
         return result;
     }
+
```

Comment:
maybe leave just uct test, because this one does the same and kinda black box?

Reply:
let's remove

Reply:
done